source/features/repo-age.tsx

```diff
@@ -80,6 +82,21 @@ async function init(): Promise<void> {
 		.replace(/^an?/, '1')
 		.split(' ');
 
+	const secondSidebarSection = await elementReady('.repository-content .BorderGrid-row + .BorderGrid-row');
```

Comment:
We should start marking these selectors so they can be all found and replaced once the issue is fixed.
```suggestion
	// TODO: simplify selector after https://github.com/sindresorhus/element-ready/issues/29
	const secondSidebarSection = await elementReady('.repository-content .BorderGrid-row + .BorderGrid-row');
```